src/Features/Core/Portable/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsService.cs

```diff
@@ -27,6 +29,24 @@ public Task<Document> RemoveUnnecessaryImportsAsync(Document document, Cancellat
             return GetUnnecessaryImports(model, root, predicate: null, cancellationToken: cancellationToken).CastArray<SyntaxNode>();
         }
 
+        protected SyntaxToken StripNewLines(Document document, SyntaxToken token)
```

Comment:
💭 I'm not a fan of passing `document` as an argument when the only thing it's used for is a call to `GetLanguageService`. However, I don't have an alternative solution right now, and this is a `protected` method so I don't think there's anything that needs to be done at this point.